app/views/roadmap/index.html.erb

```diff
@@ -0,0 +1,116 @@
+<% content_for :title, t('roadmap.page_title') %>
+<% content_for :extra_headers do %>
+  <meta name="description" content="<%= t('roadmap.hero.description') %>" />
+<% end %>
+
+<div class="govuk-width-container">
+  <main id="content" class="govuk-main-wrapper">
+    <section class=" govuk-!-padding-bottom-6 govuk-!-margin-bottom-6 govuk-roadmap-section">
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <%= render "govuk_publishing_components/components/title", title: t('roadmap.hero.heading'), margin_top: 0 %>
+          <p class="govuk-body"><%= t('roadmap.hero.description') %></p>
+        </div>
+
+        <div class="govuk-grid-column-one-third">
+          <%= render "image" %>
```

Comment:
Is it worth trying to limit the width of this image on tablet? It gets quite large compared with the rest of the content. This is at 635px wide.

<img width="600" alt="Screenshot 2021-01-28 at 12 48 13" src="https://user-images.githubusercontent.com/861310/106140821-2d08a880-6167-11eb-92c3-7edc48ba3ace.png">

Reply:
Good shout, it doesn't look amazing. 
I've limited the width to max 300px, I think that looks okay but let me know if you think otherwise.